### PR TITLE
Add phylogeny

### PR DESCRIPTION
**Pull request by sjanssen2**
rudimentary integration of validation and summary for BIOM artifacts extended by a phylogenetic tree, as necessary for SEPP.
Open questions to e.g. @antgonza :
* ~~Is it OK to add skbio here? qiime2 already implies this dependency.~~
* ~~why does the tree file not show up to be downloaded by the user?~~
* recommendations on the actual integration of info to the HTML part are welcome!

### CODE REVIEW

File: qtp_biom/summary.py [hunk 1]
```diff
@@ -13,18 +13,21 @@
...
   <body>
+    %s <!-- summarizing phylogenetic tree, if existent -->
     <iframe src="./support_files/%s" width="100%%" height="850" frameborder=0>
     </iframe>
   </body>
 </html>"""
 
 
-def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis):
+def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis, tree):
```
Comment (antgonza) [inline qtp_biom/summary.py]: Perhaps worth replacing tree for tree=None so by default it's always None. This should help needing to replace all calls of this method that we don't need to change. However, I _think_ the only call to this method is within this repo and not in many places.

File: qtp_biom/summary.py [hunk 2]
```diff
@@ -40,10 +43,38 @@ def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis):
         return (False, None,
                 "Only Qiime 2 visualization with an html index are supported")
 
+    # gather some stats about the phylogenetic tree if exists
+    summary_tree = ""
+    if tree is not None:
+        num_placements = sum([
```
Comment (antgonza) [inline qtp_biom/summary.py]: Worth replacing sum for len:
```python
In [1]: %timeit

In [2]: %timeit sum([1 for i in range(1000)])
10000 loops, best of 3: 37.5 µs per loop

In [3]: %timeit len([1 for i in range(1000)])
10000 loops, best of 3: 33.1 µs per loop

In [4]: %timeit sum([1 for i in range(100000)])
100 loops, best of 3: 4.33 ms per loop

In [5]: %timeit len([1 for i in range(100000)])
100 loops, best of 3: 3.82 ms per loop
```

Comment (sjanssen2) [reply qtp_biom/summary.py]: changed as suggested

File: qtp_biom/summary.py (hunk 1 above)
Comment (sjanssen2) [reply qtp_biom/summary.py]: excellent idea

**Comment by sjanssen2** [comment]
chnaged travis as suggested by @antgonza and it passes now :-)

**Comment by antgonza** [comment]
Code looks good but could you add tests for all the code added?

**Comment by sjanssen2** [comment]
Thanks @antgonza , we should probably have those, but I don't know how to update the test DB in a way that it contains an artefact (BIOM) with a phylogeny? Can you point me to some references?

**Comment by sjanssen2** [comment]
@antgonza : test for my code contributions are now in :-)

**Comment by sjanssen2** [comment]
awesome. Thanks!

**Pull request merged**